File: Backend/Models/Mesto.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    [Table("Mesta po rafovima")]
    public class Mesto 
    {
        [Column("ID mesta")]
        public int ID { get; set; }

        [Column("Kolicina")]
        public int Kolicina { get; set; }
        
        [Column("MaxKolicina")]
        public int MaxKolicina { get; set; }

        [Column("X")]
        public int X { get; set; }

        [JsonIgnore]
        public Raf Raf { get; set; }
    }
}

File: Frontend/mesto.js
import {Raf} from "./raf.js";
export class Mesto{
    constructor(i, kolicina, maxKolicina)
    {
        this.x=i;
        this.kolicina=1;
        this.maxKolicina=maxKolicina;
        this.miniKontejner=null;
    }
}